test-configs.yaml

```diff
@@ -692,6 +692,8 @@ device_types:
     boot_method: uboot
     dtb: 'exynos4412-odroidx2.dtb'
     flags: ['lpae']
```

Comment:
Remove this line instead of the blacklist.  The `lpae` flag means it will run boot tests with kernel builds that have LPAE in their defconfig.

I guess this flag may be replaced with pure YAML rules, but it was ported from the old device map implementation.